crml/staking/src/lib.rs

```diff
@@ -1191,7 +1196,10 @@ decl_module! {
 		#[weight = SimpleDispatchInfo::FixedNormal(5_000)]
 		fn set_invulnerables(origin, validators: Vec<T::AccountId>) {
 			ensure_root(origin)?;
-			<Invulnerables<T>>::put(validators);
+			<Invulnerables<T>>::put(validators.clone());
+			debug::print!("Set invulnerable:{:?}", validators.clone() );
+			Self::deposit_event(RawEvent::SetInvulnerables(validators.clone()) );
```

Comment:
[Optional] last clone can be removed

Reply:
`debug::print!("Set invulnerable:{:?}", validators.clone() );` does this need clone?

Reply:
Yep removed both clone()